kola/tests/podman/podman.go

```diff
@@ -58,9 +58,9 @@ func init() {
 
 // simplifiedContainerPsInfo represents a container entry in podman ps -a
 type simplifiedContainerPsInfo struct {
-	ID     string `json:"id"`
-	Image  string `json:"image"`
-	Status string `json:"status"`
+	ID     string `json:"ID"`
```

Comment:
golang's `json.Unmarshal` is case-insensitive (https://golang.org/pkg/encoding/json/#Unmarshal) so this isn't strictly necessary but if all versions of Podman are producing it in this format this is fine.

Reply:
I changed it back to lowercase since older versions were using lowercase before.